osi_object.proto

```diff
@@ -264,6 +273,56 @@ message StationaryObject
             //
             COLOR_WHITE = 10;
         }
+
+        //
+        // \brief Attributes of type emitting structure
+        //
+        message EmittingStructureAttribute
+        {
+            // This message determines the range of the wavelength and its
+            // desired number of samples.
+            //
+            repeated WavelengthData wavelength_data = 1;
+
+            // Original intensities of emitted electromagnetic wave referred to a discrete
+            // wavelength defined in WavelengthData. This maximum value is weighted based on the IntensityDistribution.
+            //
+            // Unit: W/m^2
+            //
+            // \note max_intensity_per_wavelength.size() = WavelengthData.samples_number.size()
```

Comment:
**Output from CCB 09.06.2021**
A none "equation" based definition to be added describing the fact the  number of max_intensity_per_wavelength should be equal to the number of samples defined in the WavelengthData.